packages/wonder-blocks-clickable/components/clickable-behavior.js

```diff
@@ -361,6 +360,12 @@ export default class ClickableBehavior extends React.Component<
                 // indicating that we're waiting for navigation to occur.
                 this.setState({waiting: true});
             }
+
+            if (this.props.target === "_blank") {
+                this.navigateOrReset(shouldNavigate);
```

Comment:
Shouldn't we be calling `safeWithNav` before we call this? And we don't need to set `waiting: true` in this case either. I think we can simplify this a lot by moving the target logic up to the top, like you did in `navigateOrReset`. In fact, it seems like we should just do: `if (history && !skipClientNav || this.props.target === "_blank") {` and that would probably do the right thing.

Reply:
Good call.  Not sure what I was thinking not calling `safeWithNav`.  I was thinking of checking that it was called in our tests.  I need to remember my testing ABCs (Always Be Checking). 😅